ocs/ocp.py

```diff
@@ -251,3 +252,34 @@ def wait_for_resource(
                 return True
 
         return False
+
+
+def switch_to_project(project_name):
+    """
+    Switch to another project
+
+    Args:
+        project_name (str): Name of the project to be switched to
+
+    Returns:
+        bool: True on success, False otherwise
+    """
+    log.info(f'Switching to project {project_name}')
+    cmd = f'oc project {project_name}'
+    success_msgs = [
+        f'Now using project "{project_name}"',
+        f'Already on project "{project_name}"'
+    ]
+    if any(msg in run_cmd(cmd) for msg in success_msgs):
```

Comment:
run_cmd will run twice if we are passing the project_name which we are already present

Reply:
Done